### ISSUE TITLE
TEDs

### ISSUE THREAD

**Issue by cqray1990** [JiaquanYe/TableMASTER-mmocr#38]
i use text dect model as you supply,get PSENet detection results, TableMASTER structure recognition results and MASTER textline recognition results, and use match.py to get TEDS score.
PSENet + MASTER + TableMASTER_maxlength_500:
In the validation set, the accuracy is 0.56. also is not 0.9658

i  do valiation step by step as readme refer to, can't not recurrent the results: here is my cfg in  table_inference.py:

    cfg = {
        'pse_config':'/configs/textdet/psenet/psenet_r50_fpnf_600e_pubtabnet.py',
        'master_config':'/configs/textrecog/master/master_lmdb_ResnetExtra_tableRec_dataset_dynamic_mmfp16.py',
        'structure_master_config':/configs/textrecog/master/table_master_lmdb_ResnetExtract_Ranger_0930.py',
        'pse_ckpt':'./pse_epoch_600.pth',
        'master_ckpt':'./master_epoch_6.pth',
        'structure_master_ckpt':'./epoch_16_0.7767.pth',
        'end2end_result_folder':'/mmocr_pubtabnet_recognition_0917val/work_dirs/end2end_val_result',
        'structure_master_result_folder':'/mmocr_pubtabnet_recognition_0917val/work_dirs/structure_val_result',

        'test_folder':'/pubtabnet/pubtabnet/val',
        # 'test_folder':'./smallVal10'
        'chunks_nums':chunk_nums
    }

**Comment by delveintodetail** [comment]
Visualize result of each step. Check if the text line detection model, the accuracy of the table structure recognition, the text line detection are all right.

**Comment by JiaquanYe** [comment]
> i use text dect model as you supply,get PSENet detection results, TableMASTER structure recognition results and MASTER textline recognition results, and use match.py to get TEDS score. PSENet + MASTER + TableMASTER_maxlength_500: In the validation set, the accuracy is 0.56. also is not 0.9658
> 
> i do valiation step by step as readme refer to, can't not recurrent the results: here is my cfg in table_inference.py:
> 
> ```
> cfg = {
>     'pse_config':'/configs/textdet/psenet/psenet_r50_fpnf_600e_pubtabnet.py',
>     'master_config':'/configs/textrecog/master/master_lmdb_ResnetExtra_tableRec_dataset_dynamic_mmfp16.py',
>     'structure_master_config':/configs/textrecog/master/table_master_lmdb_ResnetExtract_Ranger_0930.py',
>     'pse_ckpt':'./pse_epoch_600.pth',
>     'master_ckpt':'./master_epoch_6.pth',
>     'structure_master_ckpt':'./epoch_16_0.7767.pth',
>     'end2end_result_folder':'/mmocr_pubtabnet_recognition_0917val/work_dirs/end2end_val_result',
>     'structure_master_result_folder':'/mmocr_pubtabnet_recognition_0917val/work_dirs/structure_val_result',
>  
>     'test_folder':'/pubtabnet/pubtabnet/val',
>     # 'test_folder':'./smallVal10'
>     'chunks_nums':chunk_nums
> }
> ```

We have released the TEDS evaluation log, you can download the log file by this [link](https://drive.google.com/file/d/1PGq9Y12eMfXMWiHJMi-_SgHyO1ACxnzp/view?usp=sharing), and check each val image's TEDS score.

**Comment by cqray1990** [comment]
@JiaquanYe  i find detect is right ,but text recon is wired
![PMC3416705_010_00](https://user-images.githubusercontent.com/32585434/147411174-acea088b-30ac-4d64-af20-3d6c949c9f68.png)

**Comment by JiaquanYe** [comment]
> @JiaquanYe  i find detect is right ,but text recon is wired
> 
> ![PMC3416705_010_00](https://user-images.githubusercontent.com/32585434/147411174-acea088b-30ac-4d64-af20-3d6c949c9f68.png)
> 
> 

Maybe your alphabet is different with the text-line pretrained model. We have released the alphabet in “table_recognition/demo/alphabet”
